init/init.go

```diff
@@ -334,6 +340,68 @@ func RunInit() error {
 			}
 			return cfg, nil
 		}},
+		config.CfgFuncData{"b2d env", func(cfg *config.CloudConfig) (*config.CloudConfig, error) {
```

Comment:
This part looks complicated and difficult to maintain.
The `UdevSettle ` also runs in `cloud-init`. If we put `cloud-init` before `b2d env`, will it be simpler?